scorep/__main__.py

```diff
@@ -11,6 +12,39 @@ def _err_exit(msg):
     sys.exit(1)
 
 
+def print_help():
+    print("""
+Usage: python -m scorep [options] [--] your_program.py [args]
+
+Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed:
```

Comment:
```suggestion
Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed. Any unknown option are passed directly to 'scorep-config'.
```

+ line breaks, no not expect wide terminals